src/pudl/scripts/dbt_helper.py

```diff
@@ -470,7 +471,9 @@ def migrate_tests(table_name: str, test_config_name: str, model_name: str | None
     _write_dbt_yaml_config(schema_path, schema)
 
 
-@click.group()
+@click.group(
+    context_settings={"help_option_names": ["-h", "--help"]},
+)
```

Comment:
Since dbt_helper is a real console script now, it has to pass our console script tests, which means accepting both "-h" and "--help" which it didn't before